### DISCUSSION TITLE
Adding an AMOLED theme.

### DISCUSSION THREAD

**Discussion by Impostu** [PaulWoitaschek/Voice#1284] [Ideas]
**Is your feature request related to a problem? Please describe.**
The feature request is not related to a problem but a quality of life improvement. I mostly listen to audiobooks at bedtime and an AMOLED theme would be really helpful. 

**Describe the solution you'd like**
A theme selection option in the settings that shows options like Light/Dark/Black.

**Comment by sabi0** [comment]
Apparently the application follows the system level theme setting:
https://github.com/PaulWoitaschek/Voice/issues/1012#issuecomment-1038711563

**Reply by zkvsky** [reply]
This is not enough to enable pure black (AMOLED) themed.

**Comment by KR4U7H3R** [comment]
I like the idea. A black/dark switch option for the dark mode would be nice.

**Comment by ghost** [comment]
An AMOLED/black theme would be amazing, really hope this will get implemented

**Comment by d0ob** [comment]
Also wish for a full black theme !
